dist/scss/Components/FormGroup/style.scss

```diff
@@ -0,0 +1,46 @@
+@import './../../vars';
+
+// Normal State
+$label-Dark-Text: #4729b3;
+$input-field-BG: #fff;
+$input-field-border: 1px solid #4729b3;
+$placeholder-text: #555;
+$description-text: #555;
+
+// Hover
+$input-field-border-hover: 1px solid #4729b3;
+$transition-duration-super-short: 0.15s;
+$transition: border-color 0.15s ease-in-out;
+
+// Active
+$label-Dark-Text-active: $primary-color;
+$input-field-border-active: 1px solid $primary-color;
+$box-shadow: 0 0 6px 3px $primary-color;
+$transition-active: border-color $transition-duration-super-short ease-in-out,
```

Comment:
No need to create vars if you use them only once in the same file. The purpose of vars is to use them across different components.
So e.g. if we have
```
$box-shadow: 0 0 6px 3px $primary-color;
```

for all components and decide to change to 

```
$box-shadow: 0 0 6px 3px $secondary-color;
```

we won't have to do it in each file.

So if you could think which vars will be needed for other components as well and store those in `vars.scss` it would be great!